src/main.rs

```diff
@@ -36,7 +36,9 @@ fn main(mut req: Request) -> Result<Response, Error> {
 
     // Only permit GET requests.
     if req.get_method() != Method::GET {
-        return Ok(Response::from_body("Access denied").with_status(StatusCode::FORBIDDEN));
+        return Ok(Response::from_body("Method not allowed")
+            .with_status(StatusCode::METHOD_NOT_ALLOWED)
+            .with_header(header::ALLOW, "GET"));
```

Comment:
Sadly, I had to use a string literal here for the "GET" value instead of using the `Method::GET` enum variant, as the latter caused a compilation error, as `fastly::http::Method` does not implement the trait `fastly::convert::ToHeaderValue`, which would be useful for this any many other standard headers if it did. 

cc @acfoltzer @katie-martin-fastly

Reply:
Update: I realised I needed to add OPTIONS and then used `format!()` to work around the above issue.